demo/radio-button-basic-demos.html

```diff
@@ -5,22 +5,93 @@
         display: block;
       }
     </style>
+    <h3>Basic usage</h3>
+    <vaadin-demo-snippet id="basic-usage">
+      <template preserve-content>
+        <vaadin-radio-group label="Label" theme="vertical">
+          <vaadin-radio-button checked>Option one</vaadin-radio-button>
+          <vaadin-radio-button>Option two</vaadin-radio-button>
+          <vaadin-radio-button>Option three</vaadin-radio-button>
+        </vaadin-radio-group>
+      </template>
+    </vaadin-demo-snippet>
+
+    <h3>Disabled and disabled item</h3>
```

Comment:
I would recommend to shorten this to use "Disabled state". 
Users should be able to recognise the difference.

Reply:
Done